cloudinit/config/schema.py

```diff
@@ -937,9 +938,10 @@ def validate_cloudconfig_file(
             )
             return False
     try:
-        validate_cloudconfig_schema(
+        if not validate_cloudconfig_schema(
             cloudconfig, schema, strict=True, log_deprecations=False
-        )
+        ):
+            print("Cannot validate config, jsonschema dependency missing")
```

Comment:
Let's make this message include `Skipping schema validation` somehow and we should return False just like our network-config skip above as this function returns False when schema validation is not performed.
```suggestion
            print("Cannot validate config, jsonschema dependency missing")
            return False
```